addons/io_hubs_addon/preferences.py

```diff
@@ -165,6 +165,101 @@ def execute(self, context):
         return {'FINISHED'}
 
 
+def reload_user_components():
+    from .components.components_registry import unload_user_components, load_user_components
+    unload_user_components()
+    load_user_components()
+
+
+class HubsUserComponentsPath(bpy.types.PropertyGroup):
+    name: StringProperty(
+        name='User components path entry name',
+        description='The user components path entry name.',
+    )
+    path: StringProperty(
+        name='User components path path',
+        description='The user components path path. You can copy external components here.',
+        subtype='FILE_PATH'
+    )
+
+
+class HubsUserComponentsPathAdd(bpy.types.Operator):
+    bl_idname = "hubs_preferences.add_user_components_path"
+    bl_label = "Add user components path"
+    bl_description = "Adds a new component path entry"
+    bl_options = {'REGISTER', 'UNDO'}
+
+    path: bpy.props.StringProperty(name="User Components Path", default="")
```

Comment:
It looks like this `path` property is never passed anything and can be removed.

Reply:
Ops, yep, leftover. Remving...